charts/images.yaml

```diff
@@ -21,6 +21,16 @@ images:
   repository: quay.io/k8scsi/csi-attacher
   tag: v1.1.0
   runtimeVersion: 1.14.x
+- name: csi-attacher
+  sourceRepository: https://github.com/kubernetes-csi/external-attacher
+  repository: quay.io/k8scsi/csi-attacher
+  tag: v3.1.0
+  runtimeVersion: 1.18.x
+- name: csi-attacher
+  sourceRepository: https://github.com/kubernetes-csi/external-attacher
+  repository: quay.io/k8scsi/csi-attacher
+  tag: v3.1.0
+  runtimeVersion: 1.19.x
```

Comment:
```suggestion
- name: csi-attacher
  sourceRepository: https://github.com/kubernetes-csi/external-attacher
  repository: quay.io/k8scsi/csi-attacher
  tag: v3.1.0
  runtimeVersion: ">= 1.18"
```

Reply:
Ah, great. Didn't know this was possible :slightly_smiling_face: